.github/workflows/ci.yml

```diff
@@ -1,4 +1,7 @@
-on: push
+on:
+  push:
+  pull_request:
+    branches: [master]
```

Comment:
Should this not run on all PRs.

Reply:
switched this to now work on all pull requests